### ISSUE TITLE
Source code recipe to build the test ZIM files should be here

### ISSUE THREAD

**Issue by kelson42** [openzim/zim-testing-suite#3]
Many things are currently hard to do because we don't have these pieces of code.

**Comment by mgautierfr** [comment]
They are in https://github.com/openzim/zim-testing-suite/blob/main/scripts/create_test_zimfiles
And source data are in https://github.com/openzim/zim-testing-suite/tree/main/src

**Comment by kelson42** [comment]
@mgautierfr So we can recreate the ZIM files anytime?

**Comment by mgautierfr** [comment]
> So we can recreate the ZIM files anytime? Or at least we can now how which ZIM file has been created? Anyway all of this should be clearly documented.

Yes. We don't patch binary file manually. We always run the script `create_zim_files` (which create a base `small.zim` using zimwriterfs, and binary patch this file to generate broken zim files)
